setup.py

```diff
@@ -1,7 +1,7 @@
 import setuptools
 
-with open('README.rst') as read_file:
-    long_description = read_file.read()
+#with open('README.rst') as read_file:
+#    long_description = read_file.read()
```

Comment:
Don't worry about these commented lines. See https://github.com/dhimmel/hetio/issues/7. Will delete them after we merge this PR to avoid conflicts.